.github/workflows/integration_tests.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         . bin/_tag.sh
         echo "TAG=$(CI_FORCE_CLEAN=1 bin/root-tag)" >> $GITHUB_ENV
 
-        echo "DOCKER_REGISTRY=cr.l5d.io/linkerd" >> $GITHUB_ENV
+        echo "DOCKER_REGISTRY=ghcr.io/linkerd" >> $GITHUB_ENV
```

Comment:
So this is going to be reverted, right?

Reply:
Yes, In a separate PR before the release!